src/game/hud.cpp

```diff
@@ -1313,7 +1317,29 @@ namespace hud
         vec c(1, 1, 1);
         if(game::focus->state == CS_ALIVE && index >= POINTER_HAIR)
         {
-            if(crosshairweapons&2) c = vec::hexcolor(W(game::focus->weapselect, colour));
+            // pointer color
+            if(crosshairweapons&2) // if you selected an option that includes the weapon specific crosshair color in the menu
```

Comment:
In-line comments are just bad and annoying to read. Also, this line is *way* too long.

Reply:
now that you say it, I should probably create a bool for that crosshairweapons&2 because it's used everywhere and that way we could save that tiny bit of performance